changelog/7408.improvement.md

```diff
@@ -0,0 +1,3 @@
+[`POST /model/test/intents`](/pages/http-api#operation/testModelIntent) now returns
```

Comment:
technically breaking, but the `report` was previously unusable.

Reply:
You would be surprised at the extents some people go to use "unusable" things...

Reply:
would you consider this as blocking?

Reply:
We could make it a query arg, but I'd go for the risk here. The other format was really not usable.

Reply:
I guess the only risk was someone was parsing it somehow, or directly displaying it.